src/ophyd_async/panda/writers/_hdf_writer.py

```diff
@@ -172,6 +77,18 @@ async def open(self, multiplier: int = 1) -> Dict[str, DataKey]:
         }
         return describe
 
+    async def _update_datasets(self) -> None:
+        """
+        Load data from the datasets PV on the panda, update internal
+        representation of datasets that the panda will write.
+        """
+
+        capture_table = await self.panda_device.data.datasets.get_value()
+        self._datasets = [
+            _HDFDataset(dataset_name, dataset_name, [1], multiplier=1)
```

Comment:
@evalott100 and @coretl I don't think this is quite right, presuming there should be a prefix or something in the internal path but I'm not sure where to get it from?

Reply:
nope, the dataset name is the dataset path, you don't need a leading slash or anything.

E.g. in an existing dataset using `{field_name}.{capture}` as the dataset path
https://github.com/PandABlocks/PandABlocks-client/blob/master/examples/plot_counter_hdf.py